superset-frontend/src/SqlLab/components/SqlEditorLeftBar/SqlEditorLeftBar.test.jsx

```diff
@@ -94,7 +94,7 @@ test('table should be visible when expanded is true', async () => {
     expect(dbSelect).toBeInTheDocument();
     expect(schemaSelect).toBeInTheDocument();
     expect(dropdown).toBeInTheDocument();
-    expect(abUser).toHaveLength(2);
+    expect(abUser).toHaveLength(1);
```

Comment:
This test adds one item into a 'multi' select component and then selects it. With the new behaviour, that counts as selecting all which renders a custom tag with the total number of items selected. This test would fail because it expected the single tag to be rendered in the select component when that is no longer the expected behaviour, so I changed the test. 

Another option is to make it so that the select all functionality is disabled in the case where there is only one item in the options. Please let me know your thoughts

Reply:
I think I like the idea of disabling the "select all" when only 1 option is available.